frontend/packages/dev-console/src/components/pipelineruns/logs/LogsWrapperComponent.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { MultiStreamLogs } from './MultiStreamLogs';
+import { FirehoseResult } from '@console/internal/components/utils';
+import { PodKind } from '@console/internal/module/k8s';
+
+type LogsWrapperComponentProps = {
+  obj?: FirehoseResult<PodKind>;
+  taskName: string;
+};
+
+const LogsWrapperComponent: React.FC<LogsWrapperComponentProps> = ({ obj, taskName }) => {
+  const ref = React.useRef(obj?.data);
+  if (!_.isEmpty(obj?.data)) {
+    ref.current = obj.data;
+  }
+  return obj ? <MultiStreamLogs taskName={taskName} resource={ref.current} /> : null;
```

Comment:
Not that it matters... but why not 
```suggestion
  return ref.current ? <MultiStreamLogs taskName={taskName} resource={ref.current} /> : null;
```
Odd to put a catch around obj but ignore the catch logic when you go to rendering.